app/views/purchases/show.html.erb

```diff
@@ -43,6 +43,17 @@
               <td><%= @purchase.storage_view %></td>
               </tr>
             </table>
+            <table class="table">
+              <thead>
+              <tr>
+                <th>Comment:</th>
+              </tr>
+              </thead>
+              <tbody>
+              <tr>
+              <td><%= @purchase.comment_view %></td>
+              </tr>
+            </table>
```

Comment:
Can you fix up the spacing here? Nested elements like `tr` and `td` should be indented more than the parent.

Reply:
I fixed here [834f25f](https://github.com/rubyforgood/human-essentials/pull/4642/commits/834f25ff5aa7a5146ef5e7890e5726fd52c930d1)
I also fixed the indentation of the `tr` and `td` tags from the table above